src/service/viewer-impl.js

```diff
@@ -875,39 +885,42 @@ export class Viewer {
     if (!this.hasCapability('fragment')) {
       return Promise.resolve();
     }
-    return /** @type {!Promise} */ (this.sendMessageUnreliable_(
+    return /** @type {!Promise} */ (this.sendMessageCancelUnsent(
         'fragment', {fragment}, true));
   }
 
   /**
    * Triggers "tick" event for the viewer.
    * @param {!Object} message
+   * TODO: move this to performance-impl, and use sendMessage()
    */
   tick(message) {
-    this.sendMessageUnreliable_('tick', message, false);
+    this.sendMessageCancelUnsent('tick', message, false);
```

Comment:
This one is tricky :( In case of ticks, the first tick is the most important. How can we approach this?